cmd/anonymize/readme.md

```diff
@@ -0,0 +1,22 @@
+# Database Tool
+
+## Installation
+
+    pip install -r requirements.txt
+
+## Example usage
+
+```bash
+    # Shows available options
```

Comment:
No need for indentation I think. You could also add `%` symbol in front of the `python3` lines.